spec/controllers/catalog_controller_spec.rb

```diff
@@ -1,8 +1,6 @@
 require 'spec_helper'
 
 describe CatalogController, type: :controller do
-  routes { Rails.application.class.routes }
```

Comment:
This should be the default case. Shouldn't need to specify this in every controller test